carts-and-checkout/carts/cart-items/tax-items/delete-a-tax-item.md

```diff
@@ -0,0 +1,49 @@
+# Delete a Tax Item
+
+{% api-method method="delete" host="https://api.moltin.com" path="/v2/carts/:cartID/items/:itemID/taxes/:id" %}
+{% api-method-summary %}
+Delete by ID
+{% endapi-method-summary %}
+
+{% api-method-description %}
+
+{% endapi-method-description %}
+
+{% api-method-spec %}
+{% api-method-request %}
+{% api-method-path-parameters %}
+{% api-method-parameter name="id" type="string" required=true %}
+The ID for the Tax Item to delete
+{% endapi-method-parameter %}
+{% endapi-method-path-parameters %}
+
+{% api-method-headers %}
+{% api-method-parameter name="Authorization" type="string" required=true %}
+The Bearer token to grant access to the API
+{% endapi-method-parameter %}
+{% endapi-method-headers %}
+{% endapi-method-request %}
+
+{% api-method-response %}
+{% api-method-response-example httpCode=204 %}
```

Comment:
We actually return the taxItem, so is this httpCode not invalid?

Reply:
This is how the current implementation handles deletions.